packages/sdk/src/oracle/constants.ts

```diff
@@ -0,0 +1,29 @@
+import { ethers } from "ethers";
+
+export const MULTICALL2_ADDRESS = "0x5ba1e12693dc8f9c48aad8770482f4739beed696";
+
+export enum ChainId {
+  MAINNET = 1,
```

Comment:
Can you grab these from `@uma/common/PublicNetworks` even though they are in reverse (map chain ID to network name)?

Reply:
yea i can take a look at that. this was actually copied from the frontend constants in the across app.

Reply:
this looks really useful, but cant be imported due to incompatibility with common and the FE.  Going to keep this like this for now because i think this is going to evolve more as this gets more built out. We should though talk about adding constants like this to the SDK in general.